desktop/apps/consignments2/client/reducers.js

```diff
@@ -86,8 +68,8 @@ const initialState = {
   progressBars: {},
   resetPasswordSuccess: false,
   skipPhotoSubmission: false,
-  steps: sd && sd.CURRENT_USER ? last(stepsMapping, 3) : stepsMapping,
-  submission: {},
+  steps: [],
+  submission: sd.SUBMISSION || { signature: true, authenticity_certificate: true },
```

Comment:
Just double checking -- any security concerns here? Maybe set to `false` _until_ its set to `true` via some kind of state change?